yaqd_wright/_wright_filter_wheel.py

```diff
@@ -22,9 +21,7 @@ def __init__(self, name, config, config_filepath):
         if config["serial_port"] in WrightFilterWheel.serial_dispatchers:
             self._serial_port = WrightFilterWheel.serial_dispatchers[config["serial_port"]]
         else:
-            self._serial_port = SerialDispatcher(
-                aserial.ASerial(config["serial_port"], config["baud_rate"], timeout=0, rtscts=True)
-                )
+            self._serial_port = aserial.ASerial(config["serial_port"], config["baud_rate"], timeout=0, rtscts=True)
```

Comment:
rtscts can and timeout can be removed, those settings are specific to the thorlabs devices